functions/customgear/fn_onCustomGearTabButton.sqf

```diff
@@ -95,7 +95,7 @@ if (_isLeftSide) then {
 
             _hashKey = _hashKeyArray select _forEachIndex;
             _ctrlButton setVariable [QGVAR(hashKey), _hashKey];
-            _availableOptions = [_loadoutOptionsHash, _hashKey] call CBA_fnc_hashGet;
+            _availableOptions = _loadoutOptionsHash get _hashKey;
```

Comment:
This needs `getOrDefault [_hashKey,false]` so `_availableOptions` is defined in L100